mq/mq.go

```diff
@@ -135,6 +131,9 @@ func Keepalive(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-time.After(time.Second * KEEPALIVE_TIMEOUT):
+			if mqclient == nil || !mqclient.IsConnectionOpen() {
+				SetupMQTT(false)
+			}
```

Comment:
this is cause duplicate client conns,  at times IsConnectionOpen might return false, but client can still be connected
I think we should get avoid performing setup conn again in the code

Reply:
Removed the setupMQTT here. 
After the mq client close issue fixed, it's not necessary to re-establish connection.